sdk/samples/iot/README.md

```diff
@@ -587,101 +572,40 @@ This section provides an overview of the different samples available to run and
-
-  To interact with this sample, **you must use the Azure IoT Explorer**.
-
-  <details><summary><i>How to interact with the Plug and Play sample:</i></summary>
-  <p>
-
-    The capabilities are listed below.
-
-    <details><summary><b>Device Twin:</b></summary>
-    <p>
-
-    Two device twin properties are supported in this sample:
-    - A desired property named `targetTemperature` with a `double` value for the desired temperature.
-    - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature reached since device boot.
-    <br>
-
-    <b>To send a device twin desired property message:</b> Select your device's "Device Twin" tab in the Azure IoT Explorer. Add the property `targetTemperature` along with a corresponding value to the `desired` section of the device twin JSON. Select "Save" to update the document and send the twin message to the device.
-
-    ```json
-    "properties": {
-        "desired": {
-            "targetTemperature": 68.5,
-        }
-    }
-    ```
-
-    No other property names sent in a desired property message are supported. If any are sent, the log will report there is nothing to update.
-
-    Upon receiving a desired property message, the sample will update the twin property locally and send a reported property of the same name back to the service. This message will include a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description. You will see the following in the device twin JSON.
-
-    ```json
-    "properties": {
-        "reported": {
-            "targetTemperature": {
-              "value": 68.5,
-              "ac": 200,
-              "av": 14,
-              "ad": "success"
-            },
-            "maxTempSinceLastReboot": 74.3,
-        }
-    }
-    ```
-
-    </p>
-    </details>
+### IoT Plug and Play Sample
 
-    <details><summary><b>Direct Method (Command):</b></summary>
-    <p>
+- *Executable:* `paho_iot_pnp_sample`
 
-    One device command is supported in this sample: `getMaxMinReport`.
+  This sample connects an Azure IoT Plug and Play enabled device simulating a thermostat.  This device is described via the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). This sample demonstrates sending telemetry and properties from the device and receiving commands and writeable properties from the service.
```

Comment:
```suggestion
  This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_pnp_sample.c) connects an Azure IoT Plug and Play enabled device simulating a thermostat.  This device is described via the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). This sample demonstrates sending telemetry and properties from the device and receiving commands and writeable properties from the service.
```

Reply:
@momuno - Yup, I'll make that change as it makes sense and is in symmetry with other places.  But why are we our other samples linking to the the absolute URL?  It feels it should be the relative URL so that if someone was working on a fork of the SDK, the link would point to the correct file for the fork.

If I did this change to absolute now, for instance, then the link would be broken till this merges.

Reply:
haha, the url question!  I don't recall what or if there was a final decision made about this.  We can bring up in the C99 Friday meeting.  I was under the impression it should always be absolute if in the main branch.  Granted this is a feature branch, not main.

Reply:
Here is the info I got from Ahson on the link guidelines.  Looks like they are supposed to be absolute: https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines

Reply:
Understood.  Fixing.